src/effects/deeplearning/ModelManagerPanel.cpp

```diff
@@ -0,0 +1,809 @@
+         );
+      }
+   }
+   S.EndHorizontalLay();
+}
+
+void ModelCardPanel::PopulateShortDescription(ShuttleGui &S)
+{
+   // model description
+   // S.StartStatic(XO("Description"));
+   S.SetBorder(10);
+   mShortDescription = S.AddVariableText(
+                                       XO("%s").Format(wxString(mCard->short_description())),
+                                       false, wxLEFT);
+   S.SetBorder(10);
+   // S.EndStatic();
+}
+
+void ModelCardPanel::PopulateLongDescription(ShuttleGui &S)
+{
+   // model description
+   S.StartStatic(XO(""));
+   mLongDescription = S.AddVariableText(
+                                       XO("%s").Format(wxString(mCard->long_description())),
+                                       false, wxLEFT, 
+                                       GetSize().GetWidth() - 30);
+   S.EndStatic();
+}
+
+void ModelCardPanel::PopulateMetadata(ShuttleGui &S)
+{
+   S.StartMultiColumn(2, wxALIGN_LEFT);
+   {
+      S.AddVariableText(XO("Effect: "))
+          ->SetFont(wxFont(wxFontInfo().Bold()));
+      S.AddVariableText(XO("%s")
+                            .Format(mCard->effect_type()));
+
+      S.AddVariableText(XO("Sample Rate: "))
+          ->SetFont(wxFont(wxFontInfo().Bold()));
+      S.AddVariableText(XO("%d")
+                            .Format(mCard->sample_rate()));
+
+      std::string tagString;
+      for (auto &tag : mCard->tags())
+      {
+         if (!tagString.empty())
+            tagString = tagString +  ", " + tag;
+         else
+            tagString = tag;
+      }
+      
+      S.AddVariableText(XO("Tags: "))
+          ->SetFont(wxFont(wxFontInfo().Bold()));
+      S.AddVariableText(XO("%s")
+                            .Format(tagString));
+   }
+   S.EndMultiColumn();
+}
+
+void ModelCardPanel::PopulateInstallCtrls(ShuttleGui &S)
+{
+   DeepModelManager &manager = DeepModelManager::Get();
+
+   S.StartVerticalLay(wxCENTER, true);
+   {
+      // add a progress gauge for downloads, but hide it
+      mInstallProgressGauge = safenew wxGauge(S.GetParent(), wxID_ANY, 100); // TODO:  sizing
+      mInstallProgressGauge->SetSize(wxSize(80, 20));
+      S.AddWindow(mInstallProgressGauge);
+      mInstallProgressGauge->Show();
+
+      bool installed = manager.IsInstalled(mCard);
+      TranslatableString status = installed ? XO("installed") : XO("uninstalled");
+      mInstallStatusText = S.AddVariableText(status, true);
+
+      InstallStatus iStatus = installed ? InstallStatus::installed : InstallStatus::uninstalled;
+      wxColour statusColor = mInstallStatusColors[iStatus];
+      mInstallStatusText->SetForegroundColour(statusColor);
+
+      // TODO: do translatable strings here from the begginign
+      TranslatableString cmd = installed ? XO("Uninstall") : XO("Install");
+      mInstallButton = S.AddButton(cmd);
+
+      SetInstallStatus(installed ? InstallStatus::installed : InstallStatus::uninstalled);
+
+      mSelectButton = S.AddButton(XO("Select"));
```

Comment:
XC here for sure

Reply:
https://github.com/audacity/audacity/pull/1384/commits/e3a958eabcae9c70e5c3c94bfab6993f2ccef402